index.js

```diff
@@ -58,6 +59,7 @@ class BookCatalog {
       }
 
     }
+    console.log("loadbooks()")
```

Comment:
@muhenge @webdevmaxi 

Good work guys 👍 

but we still have one more issue to address, deleted books are not supposed to reappear after page reload.

I quote from previous reviewer:
` When books are removed using the Remove Book button, they all re-appear when the page is loaded. See the gif image below for the observed behavior.`

Reply:
I'm sorry I'm having a problem where it is actually working on my PC. How can I resolve this issue?

Reply:
Kindly double-check you push the latest commit, and sharing a videocast would be helpful.

Reply:
https://user-images.githubusercontent.com/86670011/126021762-9fe4c3c7-2e5c-4bd1-8c7e-59417f886492.mp4

I have checked that the repo is synced in case it wasn't and tested the website in Firefox and Chrome. It does work weirdly in Chrome and as you pointed out, the button to add books invalidates the fields when it shouldn't.